docs/docs/guides/developer_guides/local_env/sandbox_proving.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Sandbox PXE Proving
+tags: [sandbox, PXE]
+---
+
+The Sandbox does not have client-side proving in the PXE enabled by default. This reduces testing times and increases development speed by allowing for rapid iteration.
+
+You may want to enable client-side proving in the Sandbox to better understand how long it takes to execute Aztec transactions.
+
+:::note
+
+Running the sandbox with client-side proving is much slower and should only be used sparingly to analyze real proving times of executing private functions of a contract.
+
+:::
+
+## Config
+
+To enable client-side proving, update the docker compose file found at `~/.aztec/docker-compose.sandbox.yml` and add the following configuration settings under the `aztec:environment` scope.
+
+```yml
+BB_BINARY_PATH: /usr/src/barretenberg/cpp/build/bin/bb
```

Comment:
```suggestion
BB_BINARY_PATH: ~/.bb/bb
```

I think this should  be ~/.bb/bb which is the path where bb is installed when using bbup

Reply:
What's there works. 

Updating to your suggestion, i see this when starting the sandbox. 
![image](https://github.com/user-attachments/assets/df677be4-dd44-445a-a766-7a7a9f0102c6)

Reply:
i have bb installed 
![image](https://github.com/user-attachments/assets/7eaa22b6-93cc-4d2e-a552-135a07ac52ec)

Reply:
Oh, its supposed to be installed in the above path looking at the install script - https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/bbup/shell.ts#L52-L55

Reply:
My bad. What we have now is the correct path in the docker container.